src/include/duckdb/common/vector_operations/vector_operations.hpp

```diff
@@ -66,6 +66,32 @@ struct VectorOperations {
 	// result = A <= B
 	static void LessThanEquals(Vector &A, Vector &B, Vector &result, idx_t count);
 
+	template <class LEFT_TYPE, class RIGHT_TYPE, class OP, bool LEFT_CONSTANT, bool RIGHT_CONSTANT, bool NO_NULL,
+	          bool HAS_TRUE_SEL, bool HAS_FALSE_SEL>
+	static inline idx_t SelectFlatLoop(LEFT_TYPE *__restrict ldata, RIGHT_TYPE *__restrict rdata,
```

Comment:
Please move this back to the BinaryExecutor, the loops themselves should not be in the vector operations. If you need to use this loop somewhere else include the binary executor.